### ISSUE TITLE
DVD v1.1

### ISSUE THREAD

**Issue by Lynnjx** [EnVision-Research/DVD#25]
Hello, thank you for your excellent work in contributing to the depth estimation. I am curious how do you refine temporal consistency in DVD v1.1? Is it adding more types of datasets, modifying the model structure, or adding a temporal loss function? I would be grateful if you could answer.

**Comment by soyouthinkyoucantell** [comment]
Hi @Lynnjx ,

Yes, you're almost right. We added the Transphy3D dataset and adjusted some hyperparameters, but no loss function was added.

**Comment by Lynnjx** [comment]
> Hi [@Lynnjx](https://github.com/Lynnjx) ,
> 
> Yes, you're almost right. We added the Transphy3D dataset and adjusted some hyperparameters, but no loss function was added.

Thank you for your reply
